reactor-core/src/main/java/reactor/core/publisher/SinkEmptyMulticast.java

```diff
@@ -317,11 +177,20 @@ public void subscribe(final CoreSubscriber<? super T> actual) {
 		return Stream.of(subscribers);
 	}
 
+	@Override
+	public Object scanUnsafe(Attr key) {
+		if (key == Attr.TERMINATED) return subscribers == TERMINATED;
+		if (key == Attr.ERROR) return error;
+		if (key == Attr.RUN_STYLE) return Attr.RunStyle.SYNC;
```

Comment:
does `RUN_STYLE` make any sense here, actually?

Reply:
indeed, removed in latest commit